src/constants.js

```diff
@@ -1,8 +1,12 @@
+/**
+ * The URL of the Orbit app.
+ */
+export const ORBIT_ROOT_URL = "https://app.orbit.love";
+
 /**
  * The URL of the API root.
- * To be changed to `https://orbit.eu.ngrok.io` for local development.
  */
-export const ORBIT_API_ROOT_URL = "https://app.orbit.love";
+export const ORBIT_API_ROOT_URL = `${ORBIT_ROOT_URL}/api/v1`;
```

Comment:
❓ Why keep the API root URL if references have been changed to ORBIT_ROOT_URL instead? 
Plus, won't unchanged references now be pointing to a different URL? This concerns me

Reply:
Some references have been changed to point to `app.orbit.love` (for example links to a members’ profile, or tags, etc…), while others — API calls — still point to the `ORBIT_API_ROOT_URL`

Reply:
(Maybe I’m not understanding your concern right)

Reply:
I might be misinterpreting 🤔 

Historically, every request went through the old `ORBIT_API_ROOT_URL`, which was `https://app.orbit.love`

Now, some requests go through the new `ORBIT_API_ROOT_URL` (`https://app.orbit.love/api/v1`) and links go through the `ORBIT_ROOT_URL`, `https://app.orbit.love`

Doesn't this mean API requests have been changed to point from `https://app.orbit.love/whatever` to `https://app.orbit.love/api/v1/whatever`? Is this a problem?

Reply:
> Doesn't this mean API requests have been changed to point from https://app.orbit.love/whatever to https://app.orbit.love/api/v1/whatever? Is this a problem?

Yes, 100%! It is not a problem (or at least it shouldn’t be, and doesn’t appear to be in my local tests). The background here is that there are two ways to call an API endpoint:

- Using `https://app.orbit.love/<endpoint>` (i.e. the same endpoint you’d use in the UI) with a specific HTTP header `Accept: application/json`
- Using `https://app.orbit.love/api/v1/<endpoint>` which is the recommended way in our docs. The same header is recommended but not mandatory.

This PR changes API calls to use the “recommended” way

Reply:
OOOOH. Gotcha, thanks for explaining :D 
Hmmm.I'm hitting CORs issues when trying to authenticate even though the app change has shipped - is this something you see locally too or just my end?

<img width="2215" alt="image" src="https://github.com/orbit-love/orbit-browser-extension/assets/45462299/30270afc-c71d-4e33-a2d3-d7637982309f">

Reply:
Oh shoot—you’re absolutely right. I tested this change locally where CORS are not enabled. I’ll push an update on the app side to support these new paths.

Reply:
https://github.com/orbit-love/orbit-app/pull/4157 and [Fix: oauth calls are not under /api/v1](https://github.com/orbit-love/orbit-browser-extension/pull/58/commits/6914768a735f6b612cf54401e17bd6c320119876) should fix the issues! Thanks again.